dev-client/src/components/LastModified.tsx

```diff
@@ -22,7 +22,7 @@ export const LastModified = () => {
   const {t} = useTranslation();
   return (
     <Text variant="body2" fontStyle="italic">
-      {t('general.last_modified_by', {user: 'Sample Sam', date: '2023-08-16'})}
+      {t('general.last_modified_by', {user: 'Sample Sam', date: '08/16/2023'})}
```

Comment:
question: why is this changing? this is dummy text, and we should user's preferred short date format.
```suggestion
      {t('general.last_modified_by', {user: 'Sample Sam', date: '08/16/2023'})}
```

Reply:
just to match the mock-ups, makes it easier to identify visual differences when the text all matches up